tests/tests.py

```
@@ -672,3 +673,90 @@ def test_wildcard_slash_replacement(self):
+
+        def _create_job(slurm_extra=None):
+            mock_resources = MagicMock()
+            if slurm_extra is not None:
+                mock_resources.slurm_extra = slurm_extra
+                mock_resources.get.side_effect = lambda key, default=None: (
+                    slurm_extra if key == "slurm_extra" else default
+                )
+            else:
+                # No slurm_extra attribute
+                del mock_resources.slurm_extra
+                mock_resources.get.side_effect = lambda key, default=None: default
```

Ruff (0.12.2)

Unused lambda argument: `key`

(ARG005)